### PR TITLE
Fixing the build because of broken adoc files

### PR DESCRIPTION
**Pull request by LightGuard**

### CODE REVIEW

File: products/eap/get-started.adoc [hunk 1]
```diff
@@ -49,13 +49,11 @@ c.  Click _Add Repository_ and select the
...
 [.content-img]
-image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-1-1.png[Add
-Maven Repository]
+image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-1-1.png[Add Maven Repository]
 d.  Click Finish +
 +
 [.content-img]
-image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png[Configure
-Maven Repository]
+image://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png[Configure Maven Repository]
```
Comment (paulrobinson) [inline products/eap/get-started.adoc]: Why is this linking to an image in a PR?

Comment (paulrobinson) [inline products/eap/get-started.adoc]: Notice that http://static.jboss.org/rhd/pr/165/build/372/images/images_products_devstudio_devstudio-overview-2-1.png no longer exists

Comment (paulrobinson) [inline products/eap/get-started.adoc]: BTW, This file has the only 2 occurrences of 'static.jboss.org/rhd/pr' across the whole site code.

Comment (paulrobinson) [inline products/eap/get-started.adoc]: Looks like a separate issue as the images are missing here too: http://developers.redhat.com/products/eap/get-started/

Comment (LightGuard) [inline products/eap/get-started.adoc]: Looks like those should be fixed to use the #{cdn(blah)} notation. We'll have to look at the history to see what it was originally.

**Pull request merged**